### PR TITLE
Implicitly enable EXT_float_blend and promote it to Community Approved.

### PR DESCRIPTION
**Pull request by jdashg**
Update tests to require implicit activation accordingly.

### CODE REVIEW

**Comment by MarkCallow** [comment]
This is going to get us into the same situation we got into with float rendering where apps will rely on the implicit enabling and then fail on platforms where this the feature is no available. In this case, that is every mobile platforms. 

Why are we doing this? Since it is easy to update web applications why are we so keen not to break apps. They'll break anyway when run on mobile. We should force the authors to fix them by not implicitly enabling.

**Comment by jdashg** [comment]
We can't update apps that are no longer maintained, so we do this dance where we enable it implicitly, but should warn in the console when it's used implicitly, and use *that* to convince authors to improve the portability of their content.

**Comment by jdashg** [comment]
@kenrussell PTAL
I want to take this change in the tail end of Firefox 67 (which is merging to Beta soon) since we started doing the strict validation here in 67.

File: sdk/tests/conformance2/extensions/ext-float-blend.html [hunk 1]
```diff
@@ -28,10 +28,10 @@
     }
 
     debug("");
-    debug("Testing that float32 blending is forbidden without EXT_float_blend");
-    testExtFloatBlend(false, gl.RGBA32F);
-    testExtFloatBlendMRT(false);
-    testExtFloatBlendNonFloat32Type(false);
+    debug("Testing that float32 blending without EXT_float_blend");
```
Comment (kainino0x) [inline sdk/tests/conformance2/extensions/ext-float-blend.html]: "that float32 blending" what?

File: sdk/tests/js/tests/ext-float-blend.js [hunk 2]
```diff
@@ -29,7 +29,9 @@ void main()
 }
 `;
 
-function testExtFloatBlend(shouldBlend, internalFormat) {
+function testExtFloatBlend(internalFormat) {
+    const shouldBlend = gl.getSupportedExtensions().indexOf('EXT_float_blend') != -1;
```
Comment (kainino0x) [inline sdk/tests/js/tests/ext-float-blend.js]: These mean the browser MUST expose EXT_float_blend if it supports float blend, right?

File: extensions/EXT_float_blend/extension.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<draft href="EXT_float_blend/">
+<extension href="EXT_float_blend/">
```
Comment (kainino0x) [inline extensions/EXT_float_blend/extension.xml]: Shouldn't there be a public_webgl post about this first?

Comment (jdashg) [reply extensions/EXT_float_blend/extension.xml]: I can post about it, but I don't think there's an ordering requirement.

File: sdk/tests/js/tests/ext-float-blend.js (hunk 2 above)
Comment (jdashg) [reply sdk/tests/js/tests/ext-float-blend.js]: That's right.

File: sdk/tests/conformance2/extensions/ext-float-blend.html (hunk 1 above)
Comment (jdashg) [reply sdk/tests/conformance2/extensions/ext-float-blend.html]: oops, s/that //

File: sdk/tests/js/tests/ext-float-blend.js (hunk 2 above)
Comment (kainino0x) [reply sdk/tests/js/tests/ext-float-blend.js]: That's very strict, but it is nice. Seems ok with me.

Comment (kenrussell) [reply sdk/tests/js/tests/ext-float-blend.js]: SGTM too.

**Comment by jdashg** [comment]
Thanks!

**Pull request merged**